### ISSUE TITLE
Continued from closed #25864 ./configure and installing llvm

### ISSUE THREAD

**Issue by elonlvl** [bitcoin/bitcoin#25885]
Thanks to everyone and especially _fanquake_ for the help. Here we go... this is a bug continued from ./configure when installing `brew install llvm` i get a string of this::
```

==> llvm: stable 14.0.6, HEAD [keg-only]
Next-gen compiler infrastructure
https://llvm.org/
Not installed
From: https://github.com/Homebrew/homebrew-core/blob/HEAD/Formula/llvm.rb
License: Apache-2.0 with LLVM-exception
==> Dependencies
Build: cmake ✔, swig ✔
Required: python@3.10 ✔, libffi ✔
==> Options
--HEAD
	Install HEAD version
==> Caveats
To use the bundled libc++ please add the following LDFLAGS:
  LDFLAGS="-L/usr/local/opt/llvm/lib -Wl,-rpath,/usr/local/opt/llvm/lib"

```
llvm is keg-only, which means it was not symlinked into /usr/local,
because macOS already provides this software and installing another version in
parallel can cause all kinds of trouble.

i feel like i am getting nowhere . still unable to configure the bitcoin/bitcoin git...... its heartwrenching, but I am trying diligently and patiently to fix my issues. is there something wrong with my mac itself. it is very old 2011..... over ten years old!! very outdated. running Mac OS 10.13.6

do i need to go ahead and use` brew info llvm `'s out put of /usr/local/opt/llvm/lib or what sense it is not symlinked... ?? and just use that LDFLAG as the path? it says it can't symlink? should i use the keg's path /usr/local/opt/llvm or the other, or will this cause problems. i will get this thing worked out eventually. just need a little nudge in the right direction one more time.

**Comment by saranshisatgit** [comment]
Why are you trying to build this using `llvm` , I don't see any library recommendations in the doc of -osx.md did you try following up from there? I was able to compile on the latest mac book with M1 chip

**Comment by saranshisatgit** [comment]
This should work 

` ./configure CC=/opt/homebrew/opt/llvm/bin/clang@ CXX=/opt/homebrew/opt/llvm/bin/clang++  `

**Comment by elonlvl** [comment]
> Why are you trying to build this using `llvm` , I don't see any library recommendations in the doc of -osx.md did you try following up from there? I was able to compile on the latest mac book with M1 chip 

Llvm happens to be the default for my particular computer* I did follow up -osx.md to no avail. I was not able to compile with my 2008 All-in-one Mac 21.6 with no M1 Chip. It's quite old and I'm starting to think that's problem. I am running High Sierra 10.13.6 and it's the highest upgrade I can do on the particular machine bc of how old it is. Therefore my xcode.app is not fully working and I don't know much about compiling in the first place. Clang vs. CC vs. C++ vs. gcc@8 . It's all deconstructed sushi to me... And fish ain't my strong point. Except Mackerel it's so damn good. Mackerel box with the thin lemon slice on top. Don't even take it off for the first bite I love the zest. Anyways sorry. I'm hungry. But why can I not compile with what I got. My config.log gives me a dumb reason for not working....
